### DISCUSSION TITLE
[rejected] GET sale/offers/{offerId} - zwracanie daty wystawienia oferty

### DISCUSSION THREAD

**Discussion by PhotoSoft** [allegro/allegro-api#6033] [Suggestions - rejected]
[GET sale/offers/{offerId}](https://developer.allegro.pl/documentation/#operation/getOfferUsingGET) pozwala pobrać wszystkie parametry oferty **oprócz daty wystawienia**. Aby ją pobrać trzeba specjalnie robić dodatkowe wywołanie GET /sale/offers?offer.id={offerId}

Wnioskuję o zwracanie przez GET sale/offers/{offerId} również daty wystawienia - jedynego parametru, którego nie wiadomo dlaczego nie zwraca...

**Comment by hahyes** [comment]
Podpinam się, zmniejszyłoby to zapytania z mojej strony podczas regularnej pełnej synchronizacji o połowę - często wbijam się w limity zapytań na sekundę, bo nie chcę tego robić 10 godzin. Czas to pieniądz.
